dashboard/src/components/AppView/AppView.tsx

```diff
@@ -29,11 +31,11 @@ export interface IAppViewProps {
 }
 
 interface IAppViewState {
-  deployments: { [d: string]: IResource };
-  otherResources: { [r: string]: IResource };
-  services: { [s: string]: IResource };
-  ingresses: { [i: string]: IResource };
+  deployments: { [d: string]: IKubeItem<IResource> };
```

Comment:
Why do we need a map instead of an array? It seems that later we do `this.arrayFromState("services");`